README.md

```diff
@@ -38,6 +38,4 @@ The best way to learn Zenoh-Flow is to go through our [getting started guide](ht
 
 ## Examples
 
-We encourage you to look at the examples available in our [examples repository](https://github.com/ZettaScaleLabs/zenoh-flow-examples).
-
-🚗 If you still want more, we also ported an [Autonomous Driving Pipeline](https://github.com/ZettaScaleLabs/stunt)!
+compile the examples using "cargo build --examples" 
```

Comment:
It's the other way around: keep the lines you removed and remove the line you kept.